frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -18,36 +20,67 @@ export interface Props {
 export default class extends React.Component<Props> {
   render() {
     const { account } = this.props;
+    const stakedBalanceBN = new BN(account.locked);
```

Comment:
@kcole16 see these calculation here and give your comments. Since there is liquid balance which maybe refer to unlocked one